packages/wallet-ui/src/components/ui/organism/Header/SendModal/SendModal.view.tsx

```diff
@@ -108,9 +159,15 @@ export const SendModalView = ({ closeModal }: Props) => {
             </Network>
             <AddressInput
               label="To"
-              placeholder="Paste recipient address here"
+              placeholder="Paste recipient address or .stark name here"
               onChange={(value) => handleChange('address', value.target.value)}
+              disableValidate
+              validateError={errors.address}
+              validInput={isValidAddrField()}
             />
+            {isValidStarkName(fields.address) && resolvedAddress && (
+              <InfoText>{shortenAddress(resolvedAddress, 12)}</InfoText>
+            )}
             <SeparatorSmall />
             <MessageAlert
               variant="info"
```

Comment:
please help to update a message below
from 
"Please only enter a valid Starknet address or .stark name. Sending funds to a different network might result in permanent loss."
to 
"Please only enter a valid Starknet address. Sending funds to a different network might result in permanent loss."